### DISCUSSION TITLE
Can't output sound from PlayCover

### DISCUSSION THREAD

**Discussion by stamimy** [ExistentialAudio/BlackHole#728] [General]
Hello, I'm using Macbook Pro M1 and I tried this to record my iOS Gameplay from PlayCover app but it seems no audio is recorded. If I play Youtube video the audio is nicely outputted.

I followed tutorial to set this up from here https://www.youtube.com/watch?v=LSmM5FXzVBg
Another notes from PlayCover dev here https://github.com/PlayCover/PlayCover/discussions/283 it seems Blackhole doesn't support yet, and suggest using another (paid) tool called Loopback

Any idea what to do? Thanks

**Comment by devinroth** [comment]
That’s interesting. I’m 100% positive that it has nothing to do with BlackHole. Most likely it’s related to the audio API the developer used. In particular probably because of the multi-output. Some apps just don’t support multi-outputs.

Are you on Ventura? If so try out our new app under development called Dipper. Should solve that problem.
https://discord.com/channels/775472045865173023/1047774289480400986

**Reply by stamimy** [reply]
Yes, I'm on Ventura 13.3.1 (a) (22E772610a)

Ok I'll check the new app, Thanks

**Reply by albertusdev** [reply]
I just gave Dipper a try and the Dipper microphone works perfectly to fix these sideloaded apps from PlayCover! Thanks @devinroth 👍🏻

**Reply by egagofur** [reply]
Hallo, can i try the app? discord link has expired

**Comment by devinroth** [comment]
Dipper is on the App Store now.

**Comment by devinroth** [comment]
https://apps.apple.com/ca/app/dipper/id6450242673?mt=12?mt=12?pt=120783146&ct=Dipper.Audio&mt=8

**Comment by niladrix719** [comment]
Can we use mic using this

**Comment by devinroth** [comment]
Yes

Devin
